packages/nimble-components/src/wafer-map/modules/data-manager.ts

```diff
@@ -49,34 +49,25 @@ export class DataManager {
 
     private readonly computations: Computations;
     private readonly prerendering: Prerendering;
-    private readonly dataMap: Map<string, WaferMapDie>;
 
     public constructor(wafermap: WaferMap) {
         this.computations = new Computations(wafermap.dies, wafermap.quadrant, {
```

Comment:
Sincer we are doing changes here, maybe it would already make sense to pass in the waferMap reference directly, and deconstruct individual properties inside the Computations itself `new Computations(wafermap)`